.github/workflows/mdl.yml

```diff
@@ -0,0 +1,27 @@
+name: "Markdown Lint"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 21 * * 6"
+
+permissions:
+  contents: read
+
+jobs:
```

Comment:
mdl, I get it, but I wouldn’t if I hadn’t read the pull request title. Longer less hard to read names would be my preference.